probe-rs/src/debug/variable.rs

```diff
@@ -781,18 +783,12 @@ impl Variable {
                         format!("{}{}{:\t<indentation$}]", compound_value, line_feed, "")
                     }
                     VariableType::Struct(name)
-                        if /* name.starts_with("Some")
-                            || */ name.starts_with("Ok") 
-                            || name.starts_with("Err") =>
+                        if name.starts_with("Ok") || name.starts_with("Err") =>
```

Comment:
Why is this starts_with and not equals?

Reply:
Testing equality is defintely more accurate. Thanks :)